src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+
+public class Event extends Task {
+    private String from;
+    private String to;
```

Comment:
Maybe you can consider using more specific names for these variables too? Something like startDate and endDate might be more descriptive for others reading your code